runelite-mixins/src/main/java/net/runelite/mixins/CameraMixin.java

```diff
@@ -39,34 +39,42 @@
 	@Shadow("clientInstance")
 	static RSClient client;
 
+	@Shadow("isDrawingRegion")
+	static boolean isDrawingRegion;
+
+
+	@Shadow("SINE")
```

Comment:
If you want 2 less dependencies, these are computed and available in Perspective.SINE/COSINE